e2e/rbd.go

```diff
@@ -427,6 +440,53 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("Resize rbd-nbd PVC and check application directory size", func() {
+				kernelRelease, err := GetKernelVersion()
+				if err != nil {
+					e2elog.Failf("failed to get kernel version, error %v", err)
```

Comment:
`failed to get kernel version with error` to keep it consistent with other logging

Reply:
Sure